docs/source/guide/benefits/why_bayesian.md

```diff
@@ -0,0 +1,17 @@
+(why_bayesian)=
```

Comment:
You dont need to add these targets (and they are currently not being used). With targets a reference index is built to be used with `{ref}`, they should be used when we want to link to specific sections of a doc, or to specific bullet points in enumerations... or at the top level if the files might be moved or deleted (in which case we can add the target an alternative section that superseeded that one or things like that, sections can have multiple targets pointing to them). With `{doc}` we generate links using filenames